src/generic_ui/polymer/link.ts

```diff
@@ -0,0 +1,6 @@
+Polymer({
+  handleKey: function() {
+    this.click();
```

Comment:
can you comment on what this.click does?  I assume the ```this.fire('tap')``` invokes the on-tap callback, but what is this.click needed for?